app/controllers/application_controller.rb

```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::API
+  rescue_from ActiveRecord::RecordInvalid, ActionController::ParameterMissing, with: :show_errors_403
+  rescue_from ActiveRecord::RecordNotFound, with: :show_errors_404
+
+  def show_errors_403(exception)
+    render json: exception, status: :forbidden
```

Comment:
No, but the exception message is passed through the method with exception variable. 
Then, only the status code changed. 
Should I provide three methods, one for each exception ?

Reply:
~Then, only the status code changed~

1. You should filter your output = your `exception` is way too verbose for your client ... it's probaly only **expecting a _simple_ message to display**
2. You shoud always **provide the same kind of result for similar requests** ... ex.: try provoking multiple errors in one of your tests (ex.: creating with wrong title AND wrong description) ... and check the case ` ActiveRecord::RecordInvalid` ... isn't it an array in this case ? 

That's probably why it could be a better option to split things (and only return the message ... instead of the exception object _dump_)